docs/release-notes/1.0.0-preview/release-1.0.0-preview.md

```diff
@@ -0,0 +1,36 @@
+# ML.NET 1.0.0-preview Release Notes
+
+This release is `Release Candidate` for version `1.0.0` of [ML.NET](https://aka.ms/mlnet). We have closed our main [API project](https://github.com/dotnet/machinelearning/projects/13). The next release will be `1.0.0` and during this sprint we are focusing on improving documentation and samples. The goal is to avoid any new breaking changes going forward. One change in this release is that we have moved `IDataView` back into `Microsoft.ML` namespace based on some feedback that we received.
```

Comment:
Do we also want to call out that we are fixing bugs and addressing feedback during this time too?

Reply:
the bar we set was security issues, but we can take other proposals to ship room. I will update the sentence.